manifests/init.pp

```diff
@@ -155,6 +158,7 @@
   $user_hash          = {},
   $configure_firewall = undef,
   $install_java       = $jenkins::params::install_java,
+  $java_package_name  = undef,
```

Comment:
Yeah, there's defaults on the PL Java module, but some of those defaults aren't compatible with Jenkins. I think Jenkins requires at least 1.7 from the errors I was seeing.